### PR TITLE
TINY-6686: Lower z-index of throbber

### PR DESCRIPTION
**Pull request by TheSpyder**
Related Ticket: TINY-6686

Description of Changes:
This is a partial implementation of the ticket to help with the RTC demo. The rest of the implementation (dismissing active popups) can be done later to resolve the potential UI weirdness that edge cases will now produce.

Pre-checks:
* [x] Changelog entry added
* [x] Tests have been added (if applicable)
* [x] Branch prefixed with `feature/` for new features (if applicable)
* [x] License headers added on new files (if applicable)

Review:
* [x] Milestone set
* [x] Review comments resolved

GitHub issues (if applicable):

### CODE REVIEW

**Review by lnewson — APPROVED** [review]

File: modules/oxide/src/less/theme/globals/global-variables.less [hunk 1]
```diff
@@ -62,7 +62,7 @@
 @z-index-menu: 1150;
 @z-index-fullscreen: 1200;
 @z-index-sink: 1300;
```
Comment (spocke) [inline modules/oxide/src/less/theme/globals/global-variables.less:64]: It becomes hard to tell what order these are displayed with the number being the same thing without inspecting the dom. Maybe change this to 1250. It should be lower than the sink.

**Review by spocke — APPROVED** [review]

File: modules/oxide/src/less/theme/globals/global-variables.less (hunk 1 above)
Comment (TheSpyder) [reply modules/oxide/src/less/theme/globals/global-variables.less:64]: The throbber needs to be equal or less than the sink for the notifications to be on top... or did you mean make the throbber 1250?

Comment (spocke) [reply modules/oxide/src/less/theme/globals/global-variables.less:64]: Ahh, yes thobber just making the values show in what order things are displayed would be nice.

Comment (spocke) [reply modules/oxide/src/less/theme/globals/global-variables.less:64]: This feels like old c64 basic.

Comment (TheSpyder) [reply modules/oxide/src/less/theme/globals/global-variables.less:64]: CSS tries to hide the composition rules, but it's all too easy to get stuck in a weird dance with it

**Review by HAFRMO — APPROVED** [review]

**Review by ashfordneil — APPROVED** [review]
